ovs/lib/nodeinstallation.py

```diff
@@ -216,14 +215,19 @@ def setup_node(node_type=None, execute_rollback=False):
                     if cluster_name == new_cluster:  # Create a new OVS cluster
                         first_node = True
                         while True:
+                            master_ip = Interactive.ask_choice(NodeInstallationController.host_ips, 'Select the public IP address of {0}'.format(node_name))
```

Comment:
This changes the order of the interactive setup ... pls move the question for the public IP after the cluster_name, as we now can ask to create a new cluster, get a question for the ip and then the question for the cluster_name

This breaks the interactive setup and requires an update of the documentation too ...